### PR TITLE
feat (Dashboard): add widget Cross-check Review

### PR DESCRIPTION
**Pull request by valerydluski**
**🟢 Add `deploy` label if you want to deploy this Pull Request to staging environment**

#### 🧑‍⚖️ Pull Request Naming Convention

- Title should follow [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary)
- Do not put issue id in title
- Do not put WIP in title. Use [Draft PR](https://github.blog/2019-02-14-introducing-draft-pull-requests/) functionality
- Consider to add `area:*` label(s)

* [x] I followed naming convention rules

---

#### 🤔 This is a ...

- [x] New feature
- [ ] Bug fix
- [ ] Performance optimization
- [ ] Refactoring
- [ ] Test Case
- [ ] Documentation update
- [ ] Other

#### 🔗 Related issue link

[issue](https://github.com/rolling-scopes/rsschool-app/issues/1849)

#### 💡 Background and solution

image

#### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Database migration is added or not needed
- [x] Documentation is updated/provided or not needed
- [x] Changes are tested locally

### CODE REVIEW

File: client/src/modules/CourseManagement/components/CourseEventModal/index.tsx [hunk 1]
```diff
@@ -95,7 +95,12 @@ export function CourseEventModal({ data, onCancel, courseId, onSubmit }: Props)
       {data.event?.id ? (
         <Title level={4}>{data.event.name}</Title>
       ) : (
-        <Form.Item name="event" label="Event" rules={[{ required: true, message: 'Please select an event' }]}>
+        <Form.Item
+          name="event"
+          label="Event"
+          rules={[{ required: true, message: 'Please select an event' }]}
+          tooltip="sdfasfa"
```
Comment (I-vasilich-I) [inline client/src/modules/CourseManagement/components/CourseEventModal/index.tsx:102]: meaningful)

Comment (valerydluski) [reply client/src/modules/CourseManagement/components/CourseEventModal/index.tsx]: This was a test of attention😊

File: client/src/modules/StudentDashboard/components/AvailableReviewCard/AvailableReviewCard.tsx [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+export function AvailableReviewCard({ availableReviews, courseAlias }: Props) {
+  return (
+    <CommonCard
+      title="Cross-check [Review]"
+      content={
+        <>
+          {availableReviews.length ? (
+            <>
+              {availableReviews.map((el, index) => (
+                <Row key={el.id} gutter={[0, 6]}>
```
Comment (nataliagulko) [inline client/src/modules/StudentDashboard/components/AvailableReviewCard/AvailableReviewCard.tsx:20]: As I remember, all offsets should be divided by 4
```suggestion
                <Row key={el.id} gutter={[0, 8]}>
```

File: client/src/modules/StudentDashboard/components/AvailableReviewCard/AvailableReviewCard.tsx [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+                  <Link target={'_blank'} href={`./cross-check-review?course=${courseAlias}&taskId=${el.id}`}>
```
Comment (nataliagulko) [inline client/src/modules/StudentDashboard/components/AvailableReviewCard/AvailableReviewCard.tsx:21]: Should not be every element inside Row wrapped in Col?

**Review by nataliagulko — APPROVED** [review]

File: client/src/modules/StudentDashboard/components/AvailableReviewCard/AvailableReviewCard.tsx (hunk 3 above)
Comment (valerydluski) [reply client/src/modules/StudentDashboard/components/AvailableReviewCard/AvailableReviewCard.tsx:21]: not need

File: client/src/modules/StudentDashboard/components/AvailableReviewCard/AvailableReviewCard.tsx (hunk 2 above)
Comment (valerydluski) [reply client/src/modules/StudentDashboard/components/AvailableReviewCard/AvailableReviewCard.tsx]: fixed

**Pull request merged**